components/listings/ListingDetails.tsx

```
@@ -283,6 +283,118 @@ export const ListingDetails: React.FC<ListingDetailsProps> = ({ listing }) => {
               </div>
             </section>
 
+            {/* Full Description - Only show if available */}
+            {isValidString(listing.fullDescription) && (
+              <section className="bg-white dark:bg-gray-800 rounded-xl p-4 sm:p-6 shadow-lg">
+                <h2 className="text-lg sm:text-xl font-semibold text-gray-900 dark:text-white mb-4">
+                  About the Role
+                </h2>
+                <div className="prose dark:prose-invert max-w-none text-gray-700 dark:text-gray-200"
+                  dangerouslySetInnerHTML={{ __html: listing.fullDescription || '' }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)